products/compute/api.yaml

```diff
@@ -2076,6 +2076,7 @@ objects:
         required: true
   - !ruby/object:Api::Resource
     name: 'Network'
+    exclude: true
```

Comment:
Is this change included by mistake?

Reply:
Yes, I used that for testing purpose, I will remove this.